.github/workflows/main.yml

```diff
@@ -76,24 +80,32 @@ jobs:
           python: '3.10'
           allow_failure: false
 
-        - name: py39-dj32-postgres-xdist-coverage
+        - name: py39-dj41-mysql_innodb-coverage
           python: 3.9
           allow_failure: false
 
         - name: py39-dj40-mysql_innodb-coverage
           python: 3.9
           allow_failure: false
 
-        - name: py38-dj32-sqlite-xdist-coverage
+        - name: py39-dj32-mysql_innodb-xdist-coverage
+          python: 3.9
+          allow_failure: false
+
+        - name: py38-dj41-sqlite-xdist-coverage
           python: 3.8
           allow_failure: false
 
         - name: py38-dj40-sqlite-xdist-coverage
           python: 3.8
           allow_failure: false
 
-        - name: py39-djmain-sqlite-coverage
-          python: 3.9
+        - name: py38-dj32-sqlite-xdist-coverage
+          python: 3.8
+          allow_failure: false
+
+        - name: py310-djmain-sqlite-coverage
+          python: '3.10'
```

Comment:
Shouldn't this be a float, not a string?

Reply:
No because then it will try to install Python 3.1, which is long dead